app/build.gradle.kts

```diff
@@ -98,10 +97,7 @@ dependencies {
     implementation(libs.credentials.play.services.auth)
     implementation(libs.googleid)
     implementation(libs.play.services.auth)
-
-    implementation(libs.navigation.fragment)
-    implementation(libs.navigation.ui)
-    implementation(libs.recyclerview)
```

Comment:
The build error you're getting is most likely due to these dependencies being accidentally removed during merge.